tla/consensus/ccfraft.tla

```diff
@@ -898,6 +898,9 @@ AdvanceCommitIndex(i) ==
 CheckQuorum(i) ==
     \* Check node is a leader (and therefore has not completed retirement)
     /\ leadershipState[i] = Leader
+    \* There must be an active config which has a replica which is not this node
```

Comment:
In other words, there must be backups, if only on the horizon, for it to make sense to check quorum at all.